src/utils/randomWidth.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Get random number.
```

Comment:
You are a liar 😣. It's not a number but a string and a special one on top of that